.browserlistrc

```diff
@@ -0,0 +1 @@
+> 1.5%
```

Comment:
This may have to be adjusted - it only gives us 77.96% coverage 
https://browserl.ist/?q=%3E+1.5%25

I understand we are just officially wanting to promise anything with greater than 1.5% usage, but if we can squeeze in a few outliers with little to no effort, I think it may be a good idea.

I'm currently using:
```
"browserslist": [
		"defaults",
		"last 2 versions",
		"not dead",
		"ie 11"
	]
```

https://browserl.ist/?q=defaults%2C+last+2+versions%2C+not+dead%2C+ie+11

This is the same as defaults really (90.49% coverage), but just adds some extra conditions (like requiring IE11 in case that gets dropped from defaults).

Reply:
I probably shouldn't change the existing supported browser list in this pull request, let's keep it as separate issue and pull request. Going to revert back to previous `last 6 versions` instead.